src/reducer.ts

```diff
@@ -23,7 +25,8 @@ const initialState: State = {
   markers: [],
   arrows: [],
   lineAnnotations: {},
-  colors: ['lightblue', 'lightgreen', 'gold', 'pink'],
+  colors: colors as unknown as Color[],
```

Comment:
Whats the as `unknown` ?

Reply:
in order for TypeScript to not treat the `const colors` as a string array I had to add `as const` to it (`src/colors.ts:21`), but that means that its type is `readonly ["pink", "purple", ...]`, and for some reason that counts as a type error here because that type doesn't match `Color[]`.

I'd want to just write it as is, or with `as Color[]`, but TS says those types don't overlap. `as unknown` erases the previous type (it's like a more typesafe `any`) and then `as Color[]` allows us to override that. it's a bit silly.